cli/tests/integrations/test_package.py

```diff
@@ -207,9 +276,6 @@ def test_install(zk_znode):
     wait_for_service('chronos')
     _uninstall_chronos()
     watch_all_deployments()
-    services = get_services(args=['--inactive'])
-    assert len([service for service in services
-                if service['name'] == 'chronos']) == 0
```

Comment:
It was slowing down the test a lot and we already do the watch after the uninstall. I can add it back in.